### DISCUSSION TITLE
Showroom

### DISCUSSION THREAD

**Discussion by Feuer-sturm** [Thyraz/Sonos-Kids-Controller#58] [Show and tell]
How does your housing of your Sonos-Kids-Controller look like? 

Use this discussion to show your housing of your project :-)

**Comment by Feuer-sturm** [comment]
I just started with the first prototype of my housing and I will update this comment with the next steps and final version.

![grafik](https://user-images.githubusercontent.com/25422320/149953831-feeb9b7b-d242-4c96-934e-d00d2907c35b.png)

Today I finalized the new case for the display and the controller:
![grafik](https://user-images.githubusercontent.com/25422320/153726006-42bd6eed-7e4d-4bc7-b2d8-fc24821ff34e.png)

**Reply by harteknut** [reply]
Here's my box, very similar to Thyraz' original version. I just lowered the display with a router to reduce the step at the front.
![image](https://user-images.githubusercontent.com/16650972/149962721-d7941e85-cf87-444c-9a63-77c2482a14c9.jpeg)
![image](https://user-images.githubusercontent.com/16650972/149962753-a96d49e0-bafe-47c2-acd6-9b444a3e7620.jpeg)

**Comment by shingoo79** [comment]
We built a Lego housing ;)

![IMG_3808](https://user-images.githubusercontent.com/1671915/150635882-ad05b2de-7e8d-4ace-af87-f49a0e21fffc.jpeg)

![C80C9E82-9A76-4FB3-842E-B614578CDBB0](https://user-images.githubusercontent.com/1671915/150635900-9abad0ec-7aa0-4949-b187-bb5fdb827fb9.jpg)
